src/Oscoin/Time/Chrono.hs

```diff
@@ -0,0 +1,42 @@
+{-- | Little helpers geared towards making obvious in the types
+      how blocks are ordered.
+--}
```

Comment:
Could you add a more detailed description and examples how to use this?

Reply:
Aye, was in the back of my head as a polishing step, thanks for keeping me honest! 👍